roles/powervs_prepare_sles_sap/tasks/main.yml

```diff
@@ -119,63 +122,64 @@
 
 # Reinforce changes in current boot
 - name: Restart network
-  service:
+  ansible.builtin.service:
     name: network
     state: restarted
 
 # following task checked for SAPTUNE profile on system and apply {{ sap_solution }} profile if not applied. Also check for saptune rpm installation
 - name: Check saptune package installation
-  zypper:
+  community.general.zypper:
```

Comment:
Check if it is community.general ? As we are not installing it, Must be other zypper command, PLease verify @AnaBil-k